packages/editor/src/client/blocks/richTextBlock.tsx

```diff
@@ -142,6 +180,19 @@ function renderElement({attributes, children, element}: RenderElementProps) {
     case BlockFormat.ListItem:
       return <li {...attributes}>{children}</li>
 
+    case BlockFormat.Table:
+      return (
+        <table>
+          <tbody {...attributes}>{children}</tbody>
```

Comment:
Was it a conscious decision to use only `tbody` and to leave out `thead` and `tfoot` ?

Reply:
Was intentionally. As the content can be styled inside the cells, I would leave it to the user to style a row differently (eg toprow bold). I thought of the table as a basic (inline-) structuring element. As @roosnic1 told me, it would be used for tsüri.ch as an element to flexibly combine pictures and text. But, I actually don't know whether not having a thead could imply any problems on some browsers?